lib/tasks/broadcasts.rake

```diff
@@ -0,0 +1,6 @@
+desc "Send Welcome Notifications"
+task broadcast_welcome_notification_flow: :environment do
```

Comment:
Not sure if we're doing this elsewhere, but I think we should include a comment about the cadence at which this task should be called (once every hour)

Reply:
Added on line 3